src/app/clinic/clinic_facade.hpp

```diff
@@ -13,18 +14,22 @@ class Clinic
     inline static std::vector<std::shared_ptr<Patient>> patients_;
     inline static std::vector<std::shared_ptr<Visit>> visits_;
     inline static std::vector<std::shared_ptr<Receptionist>> receptionists_;
+    inline static std::shared_ptr<Calendar> calendar_;
 
   public:
     static void appendDoctor(const std::shared_ptr<Doctor>& doctor);
     static void appendPatient(const std::shared_ptr<Patient>& patient);
     static void appendVisit(const std::shared_ptr<Visit>& visit);
     static void appendReceptionist(const std::shared_ptr<Receptionist>& receptionist);
+    static void addCalendar(const std::shared_ptr<Calendar>& calendar);
```

Comment:
I think it's not a good idea. It does look like we are going to make a few calendars but we dont't need it.